config/routes.rb

```diff
@@ -3,6 +3,8 @@
 
 
   post 'githook' => 'githook#index'
+  get 'githook/dev' => 'githook#dev'
+  get 'githook/master' => 'githook#master'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.